spec/fixtures/base_transformer/output/datapackage.json

```diff
@@ -0,0 +1 @@
ostal_code","type":"integer","format":"default"},{"name":"country","type":"any","format":"default"}],"missingValues":[""]},"profile":"tabular-data-resource","encoding":"utf-8"},{"format":"csv","mediatype":"text/csv","name":"organizations","path":"tmp/datapackage/data/organizations.csv","schema":{"fields":[{"name":"id","type":"integer","format":"default"},{"name":"name","type":"any","format":"default"},{"name":"alternate_name","type":"any","format":"default"},{"name":"description","type":"any","format":"default"},{"name":"email","type":"any","format":"default"},{"name":"url","type":"any","format":"default"},{"name":"tax_status","type":"any","format":"default"},{"name":"tax_id","type":"any","format":"default"},{"name":"year_incorporated","type":"any","format":"default"},{"name":"legal_status","type":"any","format":"default"}],"missingValues":[""]},"profile":"tabular-data-resource","encoding":"utf-8"},{"format":"csv","mediatype":"text/csv","name":"services_at_location","path":"tmp/datapackage/data/services_at_location.csv","schema":{"fields":[{"name":"id","type":"any","format":"default"},{"name":"service_id","type":"integer","format":"default"},{"name":"location_id","type":"integer","format":"default"},{"name":"description","type":"any","format":"default"}],"missingValues":[""]},"profile":"tabular-data-resource","encoding":"utf-8"},{"format":"csv","mediatype":"text/csv","name":"locations","path":"tmp/datapackage/data/locations.csv","schema":{"fields":[{"name":"id","type":"integer","format":"default"},{"name":"organization_id","type":"integer","format":"default"},{"name":"name","type":"any","format":"default"},{"name":"alternate_name","type":"any","format":"default"},{"name":"description","type":"any","format":"default"},{"name":"transportation","type":"any","format":"default"},{"name":"latitude","type":"any","format":"default"},{"name":"longitude","type":"any","format":"default"}],"missingValues":[""]},"profile":"tabular-data-resource","encoding":"utf-8"},{"format":"csv","mediatype":"text/csv","name":"services","path":"tmp/datapackage/data/services.csv","schema":{"fields":[{"name":"id","type":"integer","format":"default"},{"name":"organization_id","type":"integer","format":"default"},{"name":"program_id","type":"any","format":"default"},{"name":"name","type":"any","format":"default"},{"name":"alternate_name","type":"any","format":"default"},{"name":"description","type":"any","format":"default"},{"name":"url","type":"any","format":"default"},{"name":"email","type":"any","format":"default"},{"name":"status","type":"any","format":"default"},{"name":"interpretation_services","type":"any","format":"default"},{"name":"application_process","type":"any","format":"default"},{"name":"wait_time","type":"any","format":"default"},{"name":"fees","type":"any","format":"default"},{"name":"accreditations","type":"any","format":"default"},{"name":"licenses","type":"any","format":"default"}],"missingValues":[""]},"profile":"tabular-data-resource","encoding":"utf-8"}]}
```

Comment:
Would like to pretty print this but didn't know how to get the tests to ignore whitespace when comparing this with the actual output file.